manual/src/main/scala/pl/metastack/metarouter/manual/Examples.scala

```diff
@@ -20,42 +22,27 @@ object Examples extends SectionSupport {
   }
 
   section("map") {
-    case class Details(userId: Int)
-    val details = Router.route[Details](Root / "details" / Arg[Int])
     Router.parse(details, "/details/42")
   }
 
   section("parse") {
-    case class Details(userId: Int)
-    case class UserInfo(user: String, details: Boolean)
-
-    val details  = Router.route[Details](Root / "details" / Arg[Int])
-    val userInfo = Router.route[UserInfo](Root / "user" / Arg[String] / Arg[Boolean])
-
-    val routes = Router.create(details).orElse(userInfo)
-    routes.parse("/user/hello/false")
-  }
-
-  case class Details(userId: Int)
-  case class UserInfo(user: String, details: Boolean)
-
-  section("urls") {
-    val details  = Router.route[Details](Root / "details" / Arg[Int])
-    val userInfo = Router.route[UserInfo](Root / "user" / Arg[String] / Arg[Boolean])
+    val details  = Root / "details" / Arg[Int]
```

Comment:
I only redefined it for better readability, so that this snippet can be copied directly from the manual.